src/Loading/babylon.sceneLoader.ts

```diff
@@ -189,6 +189,82 @@
             }
         }
 
+        private static _CleanMatricesIndices(meshes:AbstractMesh[], skeletons:Skeleton[]): void {
+            if (!SceneLoader._CleanBoneMatrixWeights) {
+                return;
+            }
+            for (var i = 0; i < skeletons.length; i++) {
+                var skeleton = skeletons[i];
+                var noInfluenceBone = new Bone("noInfluence", skeleton, null, Matrix.Identity(), Matrix.Identity());

```

Comment:
this is done automatically by the skeleton when preparing the matrices:
https://github.com/BabylonJS/Babylon.js/blob/master/src/Bones/babylon.skeleton.ts#L241

Reply:
Ok so it is always there at skeleton.bones.length, cool!